### ISSUE TITLE
PathFigureCollectionConverter.ParseStringToPathFigureCollection not working correctly

### ISSUE THREAD

**Issue by TitusRie** [dotnet/maui#4450]
### Description

In Xamarin.Forms Paths can be generated from parsing XAML path strings.
Unfortunately, PathFigureCollectionConverter.ParseStringToPathFigureCollection in MAUI not correctly parses or displays the paths defined in the XAML path string.

The following code:
```
        var pathGeometryRectangle = new PathGeometry();
        PathFigureCollectionConverter.ParseStringToPathFigureCollection(pathGeometryRectangle.Figures, "M 5 5 H 45 V 45 H 5 Z");

        var pathGeometryCircle = new PathGeometry();
        PathFigureCollectionConverter.ParseStringToPathFigureCollection(pathGeometryCircle.Figures, "M 35 20 A 15 15 0 0 1 20 35 15 15 0 0 1 5 20 15 15 0 0 1 20 5 15 15 0 0 1 35 20 Z");
        MainPage = new ContentPage()
        {
            Content =
                new StackLayout()
                {
                    Children =
                    {
                        new Path
                        {
                            Fill = Brush.Red,
                            Stroke = Brush.Blue,
                            StrokeThickness = 3,
                            Data = pathGeometryRectangle,
                            HeightRequest = 80,
                        },

                        new Path
                        {
                            Fill = Brush.Orange,
                            Stroke = Brush.HotPink,
                            StrokeThickness = 3,
                            Data = pathGeometryCircle,
                            HeightRequest = 80,
                        },
                    }
                }
        };
```
Gives this result on WinUI:
![image](https://user-images.githubusercontent.com/69340144/152007803-401f9a45-7381-4ed5-a555-75d345732455.png)
(comparable result on Android, not tested on iOS).

Xamarin.Forms produces the correct output with the same code:
![image](https://user-images.githubusercontent.com/69340144/152009823-1e021a24-be88-4014-9101-41a3d4277d60.png)

### Steps to Reproduce

Create new MAUI App.
Paste code above in App.cs, replacing the line MainPage = new MainPage()
Run on Windows or Android

### Version with bug

Preview 12 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

Android, Windows, I was *not* able test on other platforms

### Affected platform versions

windows10.0.19041, Android API 29

### Did you find any workaround?

No workaround found

### Relevant log output

```shell
-
```

**Comment by maciej-izak** [comment]
Seems like duplicate of https://github.com/dotnet/maui/issues/2860 . MAUI without working "path" functionality seems unusable for me.

**Comment by shimingsg** [comment]
Verified Repro with Android 11 and windows 10.0.19041. Repro project available in Maui Triage Teams channel:
https://microsoft.sharepoint.com/:u:/t/Mauitriage/EVFuAUy8Fn5Jq2AIuZI6EAIBj79SXUtElw9aAluoO_KNLA?e=CTawMz

**Comment by Redth** [comment]
Closed as duplicate of #2860
